modules/mwi/mwi.c

```diff
@@ -64,6 +64,8 @@ static void notify_handler(struct sip *sip, const struct sip_msg *msg,
 		ui_output(uis, "----- MWI for %s -----\n", ua_aor(mwi->ua));
 		ui_output(uis, "%b\n", mbuf_buf(msg->mb),
 			  mbuf_get_left(msg->mb));
+		ua_event(mwi->ua, UA_EVENT_MWI_NOTIFY, NULL, "%b",
+			  mbuf_buf(msg->mb), mbuf_get_left(msg->mb));
```

Comment:
adding this event is a good thing. then the application or other modules can use it.

perhaps you could also move the `ui_output` block above to the `menu` module
and let it also handle the UA_EVENT_MWI_NOTIFY event ?